### DISCUSSION TITLE
86Box Android

### DISCUSSION THREAD

**Discussion by Slo2018** [86Box/86Box#2848] [General]
86Box for Android?

**Comment by dhrdlicka** [comment]
Theoretically possible, some team members have already compiled such builds. Personally I don't see the appeal in using a PC emulator on a mobile device, due to the considerably different input methods.

**Reply by guest123guest** [reply]
YouTubers go crazy over the Android version of Bochs running old Windows versions, so I think a version of 86Box for ARM devices will be right up their alley.

**Comment by jadasse** [comment]
I'm not sure if it would keep up with faster CPUs

**Comment by ekuesdh** [comment]
This is Good Idea. Because Limbo (QEMU fork) can't emulation Support 2D and 3D Accelerated Graphics Card (But Virtual Graphics Card except).
